topi/python/topi/x86/conv2d.py

```diff
@@ -347,9 +346,9 @@ def _declaration_conv_impl(cfg, data, kernel, strides, padding, layout, out_dtyp
                          name='output_unpack',
                          tag='conv2d_nchw',
                          attrs={'workload':
-                                    conv_arg_to_workload(data, kernel, strides,
-                                                         padding, layout,
-                                                         out_dtype)})
+                                    _conv_arg_to_workload(data, kernel, strides,
```

Comment:
If we use `@autotvm.topi_register_compute`, we don't need this. That decorator will help us to attach the workload